docs/src/running-validator/validator-reqs.md

```diff
@@ -2,35 +2,31 @@
 title: Validator Requirements
 ---
 
-## Hardware
-
-- CPU Recommendations
-  - We recommend a CPU with the highest number of cores as possible. AMD Threadripper or Intel Server \(Xeon\) CPUs are fine.
-  - We recommend AMD Threadripper as you get a larger number of cores for parallelization compared to Intel.
-  - Threadripper also has a cost-per-core advantage and a greater number of PCIe lanes compared to the equivalent Intel part. PoH \(Proof of History\) is based on sha256 and Threadripper also supports sha256 hardware instructions.
-- SSD size and I/O style \(SATA vs NVMe/M.2\) for a validator
-  - Minimum example - Samsung 860 Evo 2TB
-  - Mid-range example - Samsung 860 Evo 4TB
-  - High-end example - Samsung 860 Evo 4TB
+## Hardware Recommendations
```

Comment:
These specific specs looks decent. But @sakridge might have more thoughts than me. :)

Reply:
The Samsung 860 EVO line includes an SATA SKU.  This change set was actually inspired by a new validator being bitten by this prescription :face_with_head_bandage: